tests/db/schemas/mysql.sql

```diff
@@ -206,6 +206,6 @@ CREATE TABLE trace_tags (
 	key VARCHAR(250) NOT NULL,
 	value VARCHAR(8000),
 	request_id VARCHAR(50) NOT NULL,
-	CONSTRAINT trace_tag_pk PRIMARY KEY (key, request_id),
```

Comment:
3rd instance of the primary key constraint name being removed in the updated mysql schema